### PR TITLE
fix: change server sequence

### PR DESCRIPTION
**Pull request by aaron-steinfeld**

### CODE REVIEW

**Review by surajpuvvada — APPROVED** [review]

**Review by tim-mwangi — APPROVED** [review]

File: platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java [hunk 1]
```diff
@@ -97,12 +97,15 @@ protected void doInit() {
...
+    // Servers are processed in order. We want in process to be the last to start (so health check
+    // only returns when all other servers are up) and the last to shut down (so all other servers
+    // have drained external requests)
     this.servers =
         Stream.concat(
+                providedServers.stream(),
                 Stream.of(
                     new ConstructedServer(
-                        this.getInProcessServerName(), inProcessServerBuilder.build())),
-                providedServers.stream())
```
Comment (avinashkolluru) [inline platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java:105]: You completely removed the providedServers?
I thought you were just changing order

Comment (avinashkolluru) [reply platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java:105]: Sorry. The diff confused me.

Comment (aaron-steinfeld) [reply platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java:105]: They moved to top of stream, line 105

Comment (avinashkolluru) [reply platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java:105]: I was seeing the diff on the mobile app. It didn't show that.

**Review by ravisingal — APPROVED** [review]

**Pull request merged**